### PR TITLE
Greymatter 1.2 updates

### PR DESCRIPTION
**Pull request by dgoldstein1**
## Changes

- use larger ec2 instance, more RAM is needed
- add directions for generating keypair, if needed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,40 @@ Clone this repo onto your local machine.
 If you don't already have terraform, ansible, make, or packer installed run the `./prereqs.sh` script to install those. (Works for either Ubuntu or Mac).
 Acquire an AWS account that does not have MFA requirements.
 
+You will also need a `id_rsa` and `id_rsa.pub` key pair. You can generate them using `ssh-keygen`:
```
Comment (whenhellfreezes) [inline README.md]: Maybe have it be a recommendation. You will need an ssh key we recommend you make one using...

**Comment by whenhellfreezes** [comment]
The ram you increased is for the packer run which only creates the disk image that will later be run. The instance type for the instance create with "make apply" is configured in the terraform code. Was this your intention?

File: README.md (hunk 1 above)
Comment (whenhellfreezes) [reply README.md:12]: Nevermind coming back it is worded as a recommendation.

**Comment by chrismholmes** [comment]
Curious what requirements exist if we were to have to use this in an AWS account with MFA requirements?

**Comment by whenhellfreezes** [comment]
> Curious what requirements exist if we were to have to use this in an AWS account with MFA requirements?

Both packer and terraform would have to be configured for running with an assumed aws role. Additionally it might make sense to maybe create some sort of shell script to allow a user create that role if it is not already made.

**Comment by whenhellfreezes** [comment]
Apparently with aws-vault you can wrap the calls to terraform and packer to assume the role. Which could help with ease of use.

**Pull request merged**